cmd/analyzer/analyzer.go

```diff
@@ -239,12 +241,37 @@ func NewService(cfg *config.AnalysisConfig) (*Service, error) {
 func (a *Service) Start() {
 	a.logger.Info("starting analysis service")
 
+	var ctx context.Context
+	ctx, a.cancelAnalyzers = context.WithCancel(context.Background())
+
+	// trap Ctrl+C and call cancel on the context
+	signalChan := make(chan os.Signal, 1)
+	signal.Notify(signalChan, os.Interrupt)
```

Comment:
I've read that a lot of service managers use SIGTERM

Reply:
:+1: added

Reply:
is that pushed yet?

Reply:
Yes, Ctrl+F `SIGTERM`. And make sure you're looking at the full diff, not the diff in the context of which this thread started.

Reply:
ah ya thanks